src/components/FilterSearch.tsx

```diff
@@ -95,6 +110,8 @@ export interface FilterSearchProps {
   onSelect?: (params: OnSelectParams) => void,
   /** A function which is called when the input element's value changes. Replaces the default behavior. */
   onDropdownInputChange?: (params: OnDropdownInputChangeProps) => void,
+  /** A function which is called when the input gains focus. Replaces the default behavior. */
+  onDropdownInputFocus?: (params: OnDropdownInputFocusProps) => void,
```

Comment:
can you add/update tests similar to PR #421?

Reply:
Done